src/cc/bcc_syms.h

```diff
@@ -29,6 +29,27 @@ struct bcc_symbol {
   uint64_t offset;
 };
 
+enum bcc_stack_build_id_status {
```

Comment:
Could we just use enum and definition from the Kernel header (`"compat/linux/bpf.h"`) instead of redefining them again all over the place?

Reply:
Idea was to be backwards compatible with the kernel that doesn't support this type of reporting. The functionality may break, but build doesn't break at-least.

Reply:
This include with use the (latest) copy and we pull into BCC. See how `libbpf.h` uses the header and how we sync it with upstream periodically (Example #2100)

Reply:
Now I remember why I had to use an auxiliary type. These structures are passed between python and c++ using ctypes abstraction. Since these are defined in a kernel header, the type is never defined and hence it was not possible the define the signatures in libbcc.bcc_buildsymc apis. This worked for the normal stackmap because it was a POD type. I have removed redundant definitions in libbcc libraries as suggested.